guideline-17.md

```diff
@@ -1,7 +1,7 @@
-**17. Respect trademarks and projects.**
+<h4>17. Plugins must respect trademarks, copyrights, and project names.</h4>
 
-The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the [WordPress Foundation has trademarked the term “WordPress”](http://wordpressfoundation.org/trademark-policy/) and it is a violation to use “wordpress” in a domain name. This policy extends to plugin slugs. 
+The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the [WordPress Foundation has trademarked the term “WordPress”](http://wordpressfoundation.org/trademark-policy/) and it is a violation to use “wordpress” in a domain name.  This policy extends to plugin slugs, and we will not permit a slug to begin with another product's term.
 
-As another example, only employees of Facebook should use the slug “Facebook,” or their brand in a context such as “Facebook Dancing Sloths.” Non-employees should use a format such as “Dancing Sloths for Facebook” instead to avoid potentially misleading users into believing the plugin was developed by Facebook. Similarly, if you don't represent the "Chart.js" project, it's inappropriate to use that as the name of your plugin.
+For example only employees of Facebook should use the slug “Facebook,” or their brand in a context such as “Facebook Dancing Sloths.” Non-employees should use a format such as “Dancing Sloths for Facebook” instead to avoid potentially misleading users into believing the plugin was developed by Facebook. Similarly, if you don't represent the "Chart.js" project, it's inappropriate to use that as the name of your plugin.
```

Comment:
`“Facebook”,` instead of `“Facebook,”`, I believe.

Reply:
Punctuation goes on the inside of quotes per AP style guide - https://wmich.edu/writing/punctuation/quotationmarks

Reply:
I agree with @slaFFik: this looks weird and makes the sentence hard to read.
Per the AP style guide, this applies to quotations. However, just a name "Facebook", is not the same as a quoted phrase.